### ISSUE TITLE
Only enable Duo for certain users/groups/LDAP

### ISSUE THREAD

**Issue by sycophantic** [elie195/duo_provider#4]
Would it be possible to allow duo only for certain users/groups or even just LDAP users?

The use case is our internal users are authenticated against LDAP while our outward facing users are just local accounts.

**Comment by elie195** [comment]
Sure, I can add an option to only enforce the 2FA on non-LDAP users. You'll need to configure duo.conf with a TRUE value for LDAP_BYPASS. I'll update the README and duo.conf.example when the feature is added (need to test it a bit more).

**Comment by sycophantic** [comment]
I'd want to enforce duo for LDAP users and use another totp for non LDAP users.

**Comment by elie195** [comment]
I wanted to add that I tested with another 2FA provider (the sample "two-factor email" app) and confirmed that the user will be presented with all _enabled_ 2FA providers that are _enabled_ for the user.

So for example, the other 2FA app is enabled for all users and I configured the Duo 2FA to bypass for LDAP users. Then when I login (via LDAP), I will only be presented with an option for the other (enabled) 2FA provider.
